views/about.tmpl

```diff
@@ -0,0 +1,40 @@
+{{define "content"}}
+<div class="container-fluid current-next">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col">
+        <h1 class="text-center">About Us</h1>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="container-fluid container-padded bg-off-white">
+	<div class="container">
+    <h1 class="user-title">Welcome to URY!</h1>
+    <div class="row">
+      <div class="col-md-6 col-lg-3">
+        <h2>Who are URY?</h2>
+        <p>University Radio York (URY) is the student radio station for the University of York, broadcasting 24 hours a day during term time on 1350am across the University's Heslington Campus and worldwide via this website.</p>
```

Comment:
It'd be nice to not have to write all of this HTML boilerplate for static pages, but, again, overengineering